src/read.fs

```diff
@@ -497,14 +500,22 @@ let readExpressionText(ep: Expression): string =
         ep.getText()
 
 let readExportAssignment(ea: ExportAssignment): FsType =
-    // let var = readExpressionText ea.expression
-    // {
-    //     Namespace = []
-    //     Variable = var
-    //     Type = sprintf "%s.IExports" var
-    // }
-    // |> FsType.Import
-    FsType.None
+    printfn "kind %A" (ea.expression.kind)
+    match ea.expression.kind with
+    | SyntaxKind.Identifier ->
+        let id = ea.expression :?> Identifier
+        let exp = readExpressionText ea.expression
+        
+        printfn "export %A" exp
```

Comment:
This isn't supposed to be in this pull request, but it is harmless.  Next pull request will have this clean up.